libs/pages/cluster/src/lib/ui/page-settings-network/page-settings-network.tsx

```diff
@@ -32,6 +33,59 @@ export function PageSettingsNetwork({
     .with({ type: 'GCP_USER_PROVIDED_NETWORK' }, (f) => <GcpExistingVPC feature={f.value} />)
     .otherwise(() => null)
 
+  // Get SCW Static IP feature details
+  const staticIpFeature = cluster?.features?.find(({ id }) => id === 'STATIC_IP')
+  const isScwStaticIpEnabled =
+    cluster?.cloud_provider === 'SCW' && staticIpFeature && isClusterFeatureEnabled(staticIpFeature)
+
+  const getScwStaticIpConfig = (): { gateway_type?: string; dhcp_subnet_cidr?: string } | null => {
+    if (!staticIpFeature?.value_object) {
+      return null
+    }
+    const valueObject = staticIpFeature.value_object as any
+    const value = valueObject.value
+    if (typeof value === 'object' && value !== null && 'type' in value && value.type === 'scaleway') {
+      return {
+        gateway_type: value.gateway_type as string | undefined,
+        dhcp_subnet_cidr: value.dhcp_subnet_cidr as string | undefined,
+      }
+    }
+    return null
+  }
+
+  const scwStaticIpConfig = getScwStaticIpConfig()
+
+  // Ensure STATIC_IP feature is always shown for SCW clusters
+  const enhancedFeatures = () => {
+    if (cluster?.cloud_provider !== 'SCW') {
+      return cluster?.features || []
+    }
+
+    const features = cluster?.features || []
+    const hasStaticIp = features.some(({ id }) => id === 'STATIC_IP')
+
+    if (!hasStaticIp) {
+      // Create a default disabled STATIC_IP feature for SCW
+      return [
+        ...features,
+        {
+          id: 'STATIC_IP',
+          title: 'Static IP / Nat Gateways',
+          description:
+            'Your cluster will only be visible from a fixed number of public IP. On AWS, Nat Gateways, Private subnets and Elastic IPs will be setup.',
+          is_cloud_provider_paying_feature: true,
+          cloud_provider_feature_documentation: 'https://www.scaleway.com/en/public-gateway/',
+          value_object: {
+            type: 'BOOLEAN',
+            value: false,
+          },
+        } as any,
```

Comment:
You can't use `as any` because you lost the typage, could you cast with real type?